src/test/packer.cpp

```diff
@@ -27,6 +27,181 @@ static void ExpectAddString5(const char *pString, int Limit, const char *pExpect
 	}
 }
 
+static void ExpectAddInt(int Input, int Expected)
+{
+	CPacker Packer;
+	Packer.Reset();
+	Packer.AddInt(Input);
+	EXPECT_EQ(Packer.Data()[0], Expected);
+}
+
+static void ExpectAddExtendedInt(int Input, unsigned char *pExpected, int Size)
+{
+	CPacker Packer;
+	Packer.Reset();
+	Packer.AddInt(Input);
+	int cmp = mem_comp(Packer.Data(), pExpected, Size);
```

Comment:
```suggestion
	int Cmp = mem_comp(Packer.Data(), pExpected, Size);
```
Or even better, merge it with the next line.